mixins/RedirectMixin.js

```diff
@@ -0,0 +1,23 @@
+export default {
+    methods: {
+        checkRedirect() {
+            const redirectionMapping = this.$site.themeConfig.redirectionMapping;
```

Comment:
I suggest adding a redirect to the default page here

Reply:
Done